apps/account-worker/src/transaction_notifier/notifier.service.ts

```diff
@@ -187,6 +187,19 @@ export class TxnNotifierService
                 );
               }
               break;
+            case TransactionType.ADD_GRAPH_KEY:
```

Comment:
What if we don't want to notify via web-hooks? Doesn't seems like there are any options to disable it

Reply:
I would ultimately like to standardize across all Gateway services the following behavior for webhooks:
* clients of certain services (where appropriate) may register a global webhook to be called for all responses requiring a notification
* all requests initiating a delayed processing response may include an optional URL field for a request-specific webhook to be propagated with the request job
* completed requests will notify both the global & request-specific webhooks, if specified. If neither registered, no webhook notification will be attempted & it's not an error.
* (optional; to discuss) all services should provide a request polling endpoint for clients that prefer to poll (this, however, requires a bit more state caching in the service than is currently implemented)